grid2op/Converter/IdToAct.py

```diff
@@ -408,10 +409,7 @@ def get_gym_dict(self):
         res: :class:`gym.spaces.Dict`
             The dict
         """
-        # lazy import gym
-        from gym import spaces
-
-        res = {"action": spaces.Discrete(n=self.n)}
+        res = {"action": cls_gym._DiscreteType(n=self.n)}
```

Comment:
cls_gym is now an argument of this function that should be passed by the caller. I had to made this change because I did not want to speciliaze this function between gym / gymnasium